src/mono/wasm/wasm.proj

```diff
@@ -272,6 +270,7 @@
       <EmccExportedFunction Include="_free" />
     </ItemGroup>
     <PropertyGroup>
+      <_EmccExportedLibraryFunction>&quot;[@(EmccExportedLibraryFunction -> '%27%(Identity)%27', ',')]&quot;</_EmccExportedLibraryFunction>
```

Comment:
Add comment about `EmccExportedLibraryFunction` in the https://github.com/dotnet/runtime/blob/6706d3f04142d4a635c1266c62e4219336f4ea1e/src/mono/wasm/build/WasmApp.targets#L80 and the https://github.com/dotnet/runtime/blob/6706d3f04142d4a635c1266c62e4219336f4ea1e/src/mono/wasi/build/WasiApp.targets#L63

Reply:
it's already in `WasmApp.targets` and there is no JS in WASI